### ISSUE TITLE
Fix gcc warning

### ISSUE THREAD

**Issue by dseguin** [smcameron/space-nerds-in-space#110]
The suppressed warning in c-is-the-locale.c could be avoided with a reference cast. Here's the diff:
```
62,65c62
< #pragma GCC diagnostic push
< #pragma GCC diagnostic ignored "-Wpedantic"
< 		real_setlocale = (setlocale_prototype) dlsym(RTLD_NEXT, "setlocale");
< #pragma GCC diagnostic pop
---
> 		*(void **)(&real_setlocale) = dlsym(RTLD_NEXT, "setlocale");
```
GCC will treat that as valid ISO C. Found that in the example code in the man page for dlsym and I've used it ever since.

**Comment by smcameron** [comment]
Interesting.  Thanks!

-- steve

**Comment by smcameron** [comment]
Fixed by 3337de7428cb79ab84c82561d1b1dcd3af10a6dc
